harper-core/dictionary.dict

```diff
@@ -52867,8 +52867,62 @@ write-off/1MS
 write-up/1MS
 year-over-year/5
 
+# Phrasal verbs. These help correct the error where people write "to login" instead of "to log in"
+# For this reason the most important ones to include are the ones that have a noun counterpart.
+# Remember not to add any affixes since only the verb part at the start undergoes inflection.
+
+back up/4
```

Comment:
I've been thinking about having a special annotation flag for these. It would already be useful in linters. The one I worked on weeks ago dealing with the "backup"/"back up" problem I think finally got merged today and it would benefit already.

But I wasn't sure if the multi-word part of the dictionary would be accepted. I guess it has been. But that's why I'd put them in a separate part of the dictionary. To make best use we would really want some kind of multi-word awareness in parsing and spell checking so I hadn't pushed it any further yet.